tests/puppeteer/campaigns/functional/BO/13_shopParameters/02_orderSettings/01_orderSettings/giftOptions/01_giftOptions.js

```diff
@@ -0,0 +1,277 @@
+          `goToLoginPageFOToCheck${test.args.testIdentifier}`,
+          baseContext,
+        );
+        await this.pageObjects.homePage.goToLoginPage();
+        const pageTitle = await this.pageObjects.foLoginPage.getPageTitle();
+        await expect(pageTitle, 'Fail to open FO login page').to.contains(this.pageObjects.foLoginPage.pageTitle);
+      });
+
+      it('should sign in with default customer', async function () {
+        await testContext.addContextItem(
+          this,
+          'testIdentifier',
+          `sighInFOToCheck${test.args.testIdentifier}`,
+          baseContext,
+        );
+        await this.pageObjects.foLoginPage.customerLogin(DefaultAccount);
+        const isCustomerConnected = await this.pageObjects.foLoginPage.isCustomerConnected();
+        await expect(isCustomerConnected, 'Customer is not connected').to.be.true;
+      });
+
+      it('should go to shipping step in checkout', async function () {
+        await testContext.addContextItem(
+          this,
+          'testIdentifier',
+          `goToShippingStep${test.args.testIdentifier}`,
+          baseContext,
+        );
+        await this.pageObjects.foLoginPage.goToHomePage();
+        // Go to the first product page
+        await this.pageObjects.homePage.goToProductPage(4);
+        // Add the product to the cart
+        await this.pageObjects.productPage.addProductToTheCart();
+        // Proceed to checkout the shopping cart
+        await this.pageObjects.cartPage.clickOnProceedToCheckout();
+        // Address step - Go to delivery step
+        const isStepAddressComplete = await this.pageObjects.checkoutPage.goToDeliveryStep();
+        await expect(isStepAddressComplete, 'Step Address is not complete').to.be.true;
+      });
+
+      it(`should gift checkbox visibility be '${test.args.wantedStatus}'`, async function () {
+        await testContext.addContextItem(
+          this,
+          'testIdentifier',
+          `checkGiftVisibility${test.args.testIdentifier}`,
+          baseContext,
+        );
+        const isGiftCheckboxVisible = await this.pageObjects.checkoutPage.isGiftCheckboxVisible();
+        await expect(
+          isGiftCheckboxVisible,
+          'Gift checkbox has not the correct status',
+        ).to.equal(test.args.wantedStatus);
+      });
+
+      if (test.args.wantedStatus) {
+        it('should check gift price and tax', async function () {
+          await testContext.addContextItem(
+            this,
+            'testIdentifier',
+            `checkGiftPrice${test.args.testIdentifier}`,
+            baseContext,
+          );
+          const giftPrice = await this.pageObjects.checkoutPage.getGiftPrice();
+          await expect(giftPrice, 'Gift price is incorrect').to.equal(
+            test.args.price === 0 ? 'Free'
+              : `€${parseFloat(
+                test.args.price * (test.args.tax === 'None' ? 1 : (1 + test.args.taxValue)),
+              ).toFixed(2)}`,
```

Comment:
You use 2 decimals here, are you sure it's always the case ?

Reply:
It's default decimals value set at currency Euro
![image](https://user-images.githubusercontent.com/48441421/78225750-fd4b2e80-74ca-11ea-9089-f328c239613c.png)